Mathlib/NumberTheory/ModularForms/EisensteinSeries/Summable.lean

```diff
@@ -130,6 +146,29 @@ lemma summand_bound_of_mem_verticalStrip {k : ℝ} (hk : 0 ≤ k) (x : Fin 2 →
   exact Real.rpow_le_rpow_of_nonpos (r_pos _) (r_lower_bound_on_verticalStrip z hB hz)
     (neg_nonpos.mpr hk)
 
+lemma linear_isTheta_right (c : ℤ) (z : ℂ) :
```

Comment:
There is not `Theta` in the statement...

Reply:
Oh its a bit hidden here: `=Θ[cofinite]`